metrics/examples/kafka/kafka-connect-metrics.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: KafkaConnect
+metadata:
+  name: my-connect-cluster
+  labels:
+    app: my-connect-cluster
+spec:
+  replicas: 1
+  bootstrapServers: my-cluster-kafka-bootstrap:9092
+  config:
+    key.converter: org.apache.kafka.connect.storage.StringConverter
+    value.converter: org.apache.kafka.connect.storage.StringConverter
+    key.converter.schemas.enable: false
+    value.converter.schemas.enable: false
```

Comment:
Is the `config` section needed for the Grafan example? Otherwise, I guess that we can omit it and leave the default values.

Reply:
I originally copied the example from https://github.com/scholzj/strimzi-training/blob/master/lab-5/connect.yaml 

Will try later today without the config values and update the PR

Reply:
Hello, I just tested this without the configuration values and connect fails to start due to:

```Replication factor: 3 larger than available brokers: 1.```

So, without this configuration values, you will need at least 3 Kafka brokers to run the connect cluster successfully, which might not be wrong since all the example use 3 Kafka brokers.

It's your call, wanna keep them or remove them?

Reply:
Well, that is right, by default you need at least a 3 node Kafka cluster. But option you have there are more not fixing that. They are just changing the converters. So I think you can remove these.